apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -172,9 +199,44 @@ maybe_create_vtep(#{<<"backend">> := Backend}, #state{netlink = Pid}) ->
           ok = try_enable_ipv6(VTEPNameStr),
           {ParsedVTEPIP6, PrefixLen6} = parse_subnet(VTEPIP6),
           {ok, _} = dcos_overlay_netlink:ipaddr_replace(Pid, inet6, ParsedVTEPIP6, PrefixLen6, VTEPNameStr),
-          lager:notice("Overlay VTEP was configured, ~s => ~p", [VTEPName, Info#{ip => VTEPIP6}])
+          lager:notice("Overlay VTEP address was configured, ~s => ~p", [VTEPName, Info#{ipv6 => VTEPIP6}])
+    end.
+
+check_vtep_link(Pid, VXLan) ->
+    #{<<"vtep_name">> := VTEPName} = VXLan,
+    VTEPNameStr = binary_to_list(VTEPName),
+    case dcos_overlay_netlink:iplink_show(Pid, VTEPNameStr) of
+        {ok, [#rtnetlink{type=newlink, msg=Msg}]} ->
+            {unspec, arphrd_ether, _, _, _, LinkInfo} = Msg,
+            Match = match_vtep_link(VXLan, LinkInfo),
+            {true, Match};
+        {error, ErrorCode, ResponseMsg} ->
```

Comment:
What does that function return if there is no such ip link?

Reply:
Thats "No such device"

Reply:
Let's much only match `{ok, [#rtnetlink{}]}` and `{error, 3992977407, []}` here. All other errors should explicitly (and automatically) crash the process. We don't really need logging in "no such device" case, it's initial state of the system.

Reply:
This error_code is not standard one, but keep constant with varied interface name as I tested. It's ok to just match `not found`.